coalib/output/ConsoleInteraction.py

```diff
@@ -338,6 +340,13 @@ def print_results_formatted(log_printer,
 
             for range in result.affected_code:
                 format_args['affected_code'] = range
+
+                if file_dict:
```

Comment:
there are unit tests that pass None value as file_dict(see `tests/output/PrintFormattedResultsTest.test_default_format()`. I think that `if` is needed, isn't it ?